### PR TITLE
Added additional Metrics

### PR DESCRIPTION
**Pull request by disocverprakash**

### CODE REVIEW

File: dashboards/google-backupdr/README.md [hunk 1]
```diff
@@ -3,4 +3,5 @@
 |BackupDr Monitoring|
 |:---------------------|
 |Filename: [backupdr-monitoring.json](backupdr-monitoring.json)|
-|This dashboard has 4 charts for the related [Backup DR metrics], including metrics for storage pool utilization, vdisk count utilization, OnVault pool Utilization along with Protected Data volume.|
+|This dashboard has 6 charts for the related [Backup DR metrics], including metrics for storage pool utilization, vdisk count utilization, OnVault pool Utilization, Protected
```
Comment (varun-c) [inline dashboards/google-backupdr/README.md:6]: This link is not formatted correctly. We can either remove the brackets around Backup DR metrics so it is not a link or we can add a link that points to the metric list for the backup and dr service.

```suggestion
|This dashboard has 6 charts for the related [Backup DR metrics](https://cloud.google.com/monitoring/api/metrics_gcp#gcp-backupdr), including metrics for storage pool utilization, vdisk count utilization, OnVault pool Utilization, Protected 
```

**Review by varun-c — COMMENTED** [review]
Can you also update the metadata.yaml file in this directory to reflect the same content as the README.md?

**Comment by disocverprakash** [comment]
> Can you also update the metadata.yaml file in this directory to reflect the same content as the README.md?

Yes Updated

File: dashboards/google-backupdr/README.md (hunk 1 above)
Comment (disocverprakash) [reply dashboards/google-backupdr/README.md]: Done, removed brackets around Backup DR Metrics

**Review by varun-c — APPROVED** [review]

**Pull request merged**